Sources/TensorFlow/Operators/Image.swift

```diff
@@ -0,0 +1,339 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// A resize algorithm.
+public enum ResizeMethod {
+  /// Nearest neighbor interpolation.
+  case nearest
+  /// Bilinear interpolation.
+  case bilinear
+  /// Bicubic interpolation.
+  case bicubic
+  /// Lanczos kernel with radius 3.
+  case lanczos3
+  /// Lanczos kernel with radius 5.
+  case lanczos5
+  /// Gaussian kernel with radius 3, sigma = 1.5 / 3.0.
```

Comment:
No equals sign for consistency.
```suggestion
  /// Gaussian kernel with radius `3`, sigma `1.5 / 3.0`.
```